v3-sdk/modifying-position/src/libs/conversion.ts

```diff
@@ -0,0 +1,7 @@
+export function fromReadableAmount(amount: number, decimals: number): number {
+  return amount * Math.pow(10, decimals)
+}
+
+export function toReadableAmount(rawAmount: number, decimals: number): number {
+  return rawAmount / Math.pow(10, decimals)
+}
```

Comment:
Should grab the latest to avoid under/overflow issues

Reply:
Good call - I wanted to that and I forgot!

Reply:
Leaving this out for now - number are too small